tests/xhprof_001.phpt

```diff
@@ -1,6 +1,5 @@
 --TEST--
 XHProf: Basic Profiling Test
-Author: Kannan
```

Comment:
Author doesn't belong here, these make test output messy (and so possibly difficult to automatically process), and are completely unnecessary.